This patch makes sure that w3m or others will not pollute $HOME/

--- ./xmlto.in.orig	2011-11-29 12:49:57.000000000 +0100
+++ ./xmlto.in	2013-10-07 23:52:57.000000000 +0200
@@ -611,4 +611,6 @@
   # Extremely verbose
   BASH="${BASH} -x"
 fi
+make_temp -d temphome HOME || exit 1
+export HOME
 ${BASH} "$FORMAT" post-process || exit 1
